### PR TITLE
[Release-1.5] upgrade notice

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 1]
```diff
@@ -10,4 +10,48 @@ compatibility.  We also mention cases where backwards compatibility was
...
+
+The `istio-pilot` deployment has been removed in favor of the `istiod` deployment, which contains all functionality that Pilot once had. For backwards compatbility, there are still some references to Pilot.
+
+# Control Plane Security
+
+As part of the Istiod effort, we have changed how proxies securely communicate with the control plane. In previous versions, proxies would connect to the control plane securely when the setting `values.global.controlPlaneSecurityEnabled=true` was configured, which was the default for Istio 1.4. Each control plane component ran a sidecar with Citadel certificates, and proxies connected to Pilot over port 15011.
+
+In Istio 1.5, this is no longer the recommended or default way to connect the proxies with the control plane; instead, DNS certificates, which can be signed by Kubernetes or istiod, will be used without a sidecar. Proxies connect to pilot over port 15012.
+
+Note: despite the naming, in Istio 1.5 when `controlPlaneSecurityEnabled` is set to `false`, communication between the control plane will be secure by default.
```
Comment (ayj) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: If the control plane is already secure by default what value does setting `controlPlaneSecurityEnabled=true` provide? Is more more secure or secure in a different way?

Comment (howardjohn) [reply content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: If we could rename settings it would be called `legacySpiffeeCertificatesForXDSConnection` but we cannot. There is no benefit at all except that old 1.4 proxies will try to connect to Pilot. If you don't care about old proxies you can simplify things a bit and turn it off. Most users don't care, which is why its only in the upgrade notice

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 2]
```diff
@@ -10,4 +10,48 @@ compatibility.  We also mention cases where backwards compatibility was
...
+Previously, Citadel served two functionalities: writing certificates to secrets in each namespace, and serving secrets to the `nodeagent` over `gRPC` when SDS is used. In Istio 1.5, secrets are no longer written to each namespace. Instead, they are only served over gRPC. This functionality has been moved to the `istiod` deployment.
+
+## SDS Node Agent
+
+The `nodeagent` deployment has been removed. This functionality now exists in the Sidecar
+
+## Sidecar
+
+Previously, the sidecar was able to access certificates in two ways: through secrets mounted as files, or over SDS (through the `nodeagent` deployment). In Istio 1.5, this has been simplified. All secrets will be served over a locally run SDS server. For most users, these secrets will be fetched from the `istiod` deployment. For users with a custom CA, file mounted secrets can still be used, however, these will still be served by the local SDS server. This means that certificate rotations will no longer require Envoy to restart.
+
```
Comment (mandarjog) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
## Mixer Deprecation

`Mixer` is deprecated in Istio 1.5. The default profile in Istio 1.5 switches off `Mixer` completely. Telemetry V2 is the default in Istio 1.5 and it does not require `Mixer`.  

If you depend on specific Mixer features like out of process adapters, you may re-enable Mixer. Mixer will continue receiving bug fixes and security fixes until Istio 1.7.

Many features supported by `Mixer` have alternatives as specified in the [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) document including the [in-proxy extensions](https://github.com/istio/proxy/tree/master/extensions) based on the Wasm sandbox API.
If you rely on a Mixer feature that does not have an equivalent, we encourage you to open issues and discuss in the community.

Please check [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) notice for details.
```

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 3]
```diff
@@ -10,4 +10,59 @@ compatibility.  We also mention cases where backwards compatibility was
...
+There have been no changes to the deployment of `istio-cni`.
+
+## Pilot
+
+The `istio-pilot` deployment has been removed in favor of the `istiod` deployment, which contains all functionality that Pilot once had. For backwards compatibility, there are still some references to Pilot.
+
+# Mixer Deprecation
+
+Mixer is deprecated in Istio 1.5. The default profile in Istio 1.5 switches off `mixer` completely. Telemetry V2 is the default in Istio 1.5 and it does not require `mixer`.
```
Comment (douglas-reid) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: i'd swap `mixer` for `istio-telemetry`: "... switches off deployment of `istio-telemetry` completely. A new architecture for telemetry collection is used in 1.5, based on in-proxy telemetry generation. That architecture does not require a separate `istio-telemetry` process.

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 4]
```diff
@@ -10,4 +10,59 @@ compatibility.  We also mention cases where backwards compatibility was
...
+
+If you depend on specific Mixer features like out of process adapters, you may re-enable Mixer. Mixer will continue receiving bug fixes and security fixes until Istio 1.7.
+Many features supported by `mixer` have alternatives as specified in the [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) document including the [in-proxy extensions](https://github.com/istio/proxy/tree/master/extensions) based on the WebAssembly sandbox API.
```
Comment (douglas-reid) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: s/by `mixer`/by `istio-policy` and `istio-telemetry`/

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md (hunk 3 above)
Comment (douglas-reid) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: Maybe we want some background snuck in here?

"Mixer, the process behind the `istio-telemetry` and `istio-policy` deployments, has been deprecated with the 1.5 release. Use of `istio-policy` was disabled by default in Istio 1.3. In Istio 1.5, `istio-telemetry` is disabled by default as well."

Or something like that...

thoughts @mandarjog ?

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 5]
```diff
@@ -10,4 +10,59 @@ compatibility.  We also mention cases where backwards compatibility was
...
+
+If you rely on a Mixer feature that does not have an equivalent, we encourage you to open issues and discuss in the community.
+
+Please check [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) notice for details.
+
+# Control Plane Security
+
+As part of the Istiod effort, we have changed how proxies securely communicate with the control plane. In previous versions, proxies would connect to the control plane securely when the setting `values.global.controlPlaneSecurityEnabled=true` was configured, which was the default for Istio 1.4. Each control plane component ran a sidecar with Citadel certificates, and proxies connected to Pilot over port 15011.
+
+In Istio 1.5, this is no longer the recommended or default way to connect the proxies with the control plane; instead, DNS certificates, which can be signed by Kubernetes or Istiod, will be used without a sidecar. Proxies connect to pilot over port 15012.
```
Comment (douglas-reid) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: It might help make this clearer to non-security savvy folk if we added some more detail. *Who/What*  will use the DNS certs without a sidecar -- and how?

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 6]
```diff
@@ -10,4 +10,59 @@ compatibility.  We also mention cases where backwards compatibility was
...
+* MCP Server - the MCP server has been disabled by default. For most users, this is an implementation detail. If you do explicitly depend on this functionality, you will need to run the `istio-galley` deployment.
+* Experimental features (such as configuration analysis) - These features will require the `istio-galley` deployment.
+
+## Citadel
+
+Previously, Citadel served two functionalities: writing certificates to secrets in each namespace, and serving secrets to the `nodeagent` over `gRPC` when SDS is used. In Istio 1.5, secrets are no longer written to each namespace. Instead, they are only served over gRPC. This functionality has been moved to the `istiod` deployment.
+
+## SDS Node Agent
+
+The `nodeagent` deployment has been removed. This functionality now exists in the Sidecar
```
Comment (douglas-reid) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: missing a `.` at the end.

Do we want to call these Envoy Proxies instead of Sidecar?  https://istio.io/about/contribute/style-guide/#envoy ?

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 7]
```diff
@@ -10,4 +10,59 @@ compatibility.  We also mention cases where backwards compatibility was
...
+
+## Sidecar
```
Comment (douglas-reid) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: I might re-title this subsection to be `## Envoy Proxy Certificate Rotation Without Restart`, as that is the key take-away.

**Comment by mandarjog** [comment]
@incfly this needs to have `automtls` notes as well.

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 8]
```diff
@@ -10,4 +10,69 @@ compatibility.  We also mention cases where backwards compatibility was
...
+The `istio-pilot` deployment has been removed in favor of the `istiod` deployment, which contains all functionality that Pilot once had. For backwards compatibility, there are still some references to Pilot.
+
+## Mixer Deprecation
+
+`Mixer`, the process behind the `istio-telemetry` and `istio-policy` deployments, has been deprecated with the 1.5 release. `istio-policy` was disabled by default since Istio 1.3 and `istio-telemetry` is disabled by default in Istio 1.5.
+
+Telemetry is collected using an in-proxy extension mechanism (Telemetry V2) that does not require `Mixer`.
+
+If you depend on specific Mixer features like out of process adapters, you may re-enable Mixer. Mixer will continue receiving bug fixes and security fixes until Istio 1.7.
+Many features supported by `mixer` have alternatives as specified in the [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) document including the [in-proxy extensions](https://github.com/istio/proxy/tree/master/extensions) based on the WebAssembly sandbox API.
```
Comment (douglas-reid) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: we have `Mixer`, Mixer, and `mixer` here. do we want to pick a single way?

Comment (frankbu) [reply content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: The proper convention is to use Mixer when talking about the component, or `mixer` if referring specifically to the deployment, so in this doc I think they should all be Mixer (uppercase, no back-ticks).

**Comment by incfly** [comment]
Automatic mutual TLS now is default enabled. Traffic between sidecars are automatically configured as mutual TLS. You can disable this explicitly if you worry about the encryption overhead, `-- set values.global.mtls.auto=false` in install. For more details, see 
`https://preliminary.istio.io/docs/tasks/security/authentication/auto-mtls/`

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 9]
```diff
@@ -10,4 +10,69 @@ compatibility.  We also mention cases where backwards compatibility was
...
 familiar with the use and operation of Istio 1.4.
 
-TBD
+## Control Plane Restructuring
+
+In Istio 1.5, we have moved towards a new deployment model for the control plane, with many components consolidated. The following describes where various functionality has been moved to.
+
+### Istiod
+
+In Istio 1.5, there will be a new deployment, `istiod`. This component is the core of the control plane, and will handle configuration and certificate distribution, sidecar injection, and more.
```
Comment (johnma14) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: Shouldn't this be Istiod instead of istiod?

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 10]
```diff
@@ -10,4 +10,69 @@ compatibility.  We also mention cases where backwards compatibility was
 preserved but new behavior was introduced that would be surprising to someone
 familiar with the use and operation of Istio 1.4.
 
-TBD
+## Control Plane Restructuring
```
Comment (rcaballeromx) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: Can we include a mapping between the components and `istiod`, Envoy, etc?

Comment (howardjohn) [reply content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: If there are no glaring issues can we merge this and iterate improvements? if we keep making these megaPRs with 100s of comments it's very hard to follow

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 11]
```diff
@@ -10,4 +10,69 @@ compatibility.  We also mention cases where backwards compatibility was
...
+### Sidecar Injection
+
+Previously, sidecar injection was handled by a mutating webhook that was processed by a deployment named `istio-sidecar-injector`. In Istio 1.5, the same mutating webhook remains, but it will now point to the `istiod` deployment. All injection logic remains the same.
+
+### Galley
+
+* Configuration Validation - this functionality remains the same, but is now handled by the `istiod` deployment.
+* MCP Server - the MCP server has been disabled by default. For most users, this is an implementation detail. If you do explicitly depend on this functionality, you will need to run the `istio-galley` deployment.
+* Experimental features (such as configuration analysis) - These features will require the `istio-galley` deployment.
+
```
Comment (johnma14) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: Is there a link we can provide over here on how to enable the istio-galley deployment. I have seen links like this provided as reference in the previous upgrade notes. That might be helpful to users.

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 12]
```diff
@@ -10,4 +10,69 @@ compatibility.  We also mention cases where backwards compatibility was
...
+* Black Hole telemetry for TCP and HTTP protocols is not supported.
+* Histogram buckets are [significantly different](https://github.com/istio/istio/issues/20483) than Mixer Telemetry and cannot be changed.
+
+## Control Plane Security
+
+As part of the Istiod effort, we have changed how proxies securely communicate with the control plane. In previous versions, proxies would connect to the control plane securely when the setting `values.global.controlPlaneSecurityEnabled=true` was configured, which was the default for Istio 1.4. Each control plane component ran a sidecar with Citadel certificates, and proxies connected to Pilot over port 15011.
+
+In Istio 1.5, this is no longer the recommended or default way to connect the proxies with the control plane; instead, DNS certificates, which can be signed by Kubernetes or Istiod, will be used to connect to Istiod over port 15012.
+
+Note: despite the naming, in Istio 1.5 when `controlPlaneSecurityEnabled` is set to `false`, communication between the control plane will be secure by default.
```
Comment (johnma14) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: will this flag be removed in later versions. Looks like it doesn't matter what we set for this flag.

**Comment by mandarjog** [comment]
who is empowered to merge it ?

**Comment by howardjohn** [comment]
@ericvn ?

**Comment by mandarjog** [comment]
I am already trying to add something here and can't because history has been rewritten :-)

**Comment by howardjohn** [comment]
sorry did I revert some changes you made? I had to rebase but I thought I included everything

**Comment by fpesce** [comment]
> who is empowered to merge it ?

@mandarjog : Doc WG have approval power. If important documentation are not approved by noon, I'll probably force merge them to get the release out, and we'll update doc later for the nits.

**Comment by incfly** [comment]
I haven't seen auto mTLS is included in the notice, as I provided a sample paragraph in https://github.com/istio/istio.io/pull/6551#issuecomment-594710265

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 13]
```diff
@@ -10,4 +10,77 @@ compatibility.  We also mention cases where backwards compatibility was
...
+In Istio 1.5, there will be a new deployment, `istiod`. This component is the core of the control plane, and will handle configuration and certificate distribution, sidecar injection, and more.
+
+### Sidecar Injection
+
+Previously, sidecar injection was handled by a mutating webhook that was processed by a deployment named `istio-sidecar-injector`. In Istio 1.5, the same mutating webhook remains, but it will now point to the `istiod` deployment. All injection logic remains the same.
+
+### Galley
+
+* Configuration Validation - this functionality remains the same, but is now handled by the `istiod` deployment.
+* MCP Server - the MCP server has been disabled by default. For most users, this is an implementation detail. If you do explicitly depend on this functionality, you will need to run the `istio-galley` deployment.
```
Comment (adammil2000) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
* MCP Server - the MCP server has been disabled by default. For most users, this is an implementation detail. If you depend on this functionality, you will need to run the `istio-galley` deployment.
```

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 14]
```diff
@@ -10,4 +10,77 @@ compatibility.  We also mention cases where backwards compatibility was
...
+* Experimental features (such as configuration analysis) - These features will require the `istio-galley` deployment.
+
+### Citadel
+
+Previously, Citadel served two functionalities: writing certificates to secrets in each namespace, and serving secrets to the `nodeagent` over `gRPC` when SDS is used. In Istio 1.5, secrets are no longer written to each namespace. Instead, they are only served over gRPC. This functionality has been moved to the `istiod` deployment.
```
Comment (adammil2000) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
Previously, Citadel served two functions: writing certificates to secrets in each namespace, and serving secrets to the `nodeagent` over `gRPC` when SDS is used. In Istio 1.5, secrets are no longer written to each namespace. Instead, they are only served over gRPC. This functionality has been moved to the `istiod` deployment.
```

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 15]
```diff
@@ -10,4 +10,77 @@ compatibility.  We also mention cases where backwards compatibility was
...
-TBD
+## Control Plane Restructuring
+
+In Istio 1.5, we have moved towards a new deployment model for the control plane, with many components consolidated. The following describes where various functionality has been moved to.
+
+### Istiod
+
+In Istio 1.5, there will be a new deployment, `istiod`. This component is the core of the control plane, and will handle configuration and certificate distribution, sidecar injection, and more.
+
+### Sidecar Injection
```
Comment (adammil2000) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
### Sidecar injection
```

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 16]
```diff
@@ -10,4 +10,77 @@ compatibility.  We also mention cases where backwards compatibility was
...
+
+### CNI
+
+There have been no changes to the deployment of `istio-cni`.
+
+### Pilot
+
+The `istio-pilot` deployment has been removed in favor of the `istiod` deployment, which contains all functionality that Pilot once had. For backwards compatibility, there are still some references to Pilot.
+
+## Mixer Deprecation
```
Comment (adammil2000) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
## Mixer deprecation
```

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 17]
```diff
@@ -10,4 +10,77 @@ compatibility.  We also mention cases where backwards compatibility was
...
+Telemetry is collected using an in-proxy extension mechanism (Telemetry V2) that does not require Mixer.
+
+If you depend on specific Mixer features like out of process adapters, you may re-enable Mixer. Mixer will continue receiving bug fixes and security fixes until Istio 1.7.
+Many features supported by Mixer have alternatives as specified in the [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) document including the [in-proxy extensions](https://github.com/istio/proxy/tree/master/extensions) based on the WebAssembly sandbox API.
+
+If you rely on a Mixer feature that does not have an equivalent, we encourage you to open issues and discuss in the community.
+
+Please check [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) notice for details.
+
+### Feature Gaps between Telemetry V2 and Mixer Telemetry
```
Comment (adammil2000) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
### Feature gaps between Telemetry V2 and Mixer Telemetry
```

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 18]
```diff
@@ -10,4 +10,77 @@ compatibility.  We also mention cases where backwards compatibility was
...
+
+* Out of mesh telemetry is not supported. Some telemetry is missing if source or the destination of traffic is not sidecar injected.
+* Egress gateway telemetry is [not supported](https://github.com/istio/istio/issues/19385).
+* TCP telemetry is only supported with `mtls`.
+* Black Hole telemetry for TCP and HTTP protocols is not supported.
+* Histogram buckets are [significantly different](https://github.com/istio/istio/issues/20483) than Mixer Telemetry and cannot be changed.
+
+## Control Plane Security
```
Comment (adammil2000) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
## Control plane security
```

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 19]
```diff
@@ -10,4 +10,77 @@ compatibility.  We also mention cases where backwards compatibility was
...
+If you depend on specific Mixer features like out of process adapters, you may re-enable Mixer. Mixer will continue receiving bug fixes and security fixes until Istio 1.7.
+Many features supported by Mixer have alternatives as specified in the [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) document including the [in-proxy extensions](https://github.com/istio/proxy/tree/master/extensions) based on the WebAssembly sandbox API.
+
+If you rely on a Mixer feature that does not have an equivalent, we encourage you to open issues and discuss in the community.
+
+Please check [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) notice for details.
+
+### Feature Gaps between Telemetry V2 and Mixer Telemetry
+
+* Out of mesh telemetry is not supported. Some telemetry is missing if source or the destination of traffic is not sidecar injected.
```
Comment (adammil2000) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
* Out of mesh telemetry is not supported. Some telemetry is missing if the traffic source or destination is not sidecar injected.
```

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 20]
```diff
@@ -10,4 +10,77 @@ compatibility.  We also mention cases where backwards compatibility was
...
+
+## Control Plane Security
+
+As part of the Istiod effort, we have changed how proxies securely communicate with the control plane. In previous versions, proxies would connect to the control plane securely when the setting `values.global.controlPlaneSecurityEnabled=true` was configured, which was the default for Istio 1.4. Each control plane component ran a sidecar with Citadel certificates, and proxies connected to Pilot over port 15011.
+
+In Istio 1.5, this is no longer the recommended or default way to connect the proxies with the control plane; instead, DNS certificates, which can be signed by Kubernetes or Istiod, will be used to connect to Istiod over port 15012.
+
+Note: despite the naming, in Istio 1.5 when `controlPlaneSecurityEnabled` is set to `false`, communication between the control plane will be secure by default.
+
+## Multicluster Setup
```
Comment (adammil2000) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
## Multicluster setup
```

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md (hunk 12 above)
Comment (howardjohn) [reply content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: Yes it will likely be removed, its only for backwards compatibility

**Comment by johnma14** [comment]
@incfly I think your comments somehow got buried under all the other reviews. Apologies. Will push out another change on this file with the auto mtls comments. That needs to go in here. Thanks for bringing it up.

**Comment by incfly** [comment]
Thanks!

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 21]
```diff
@@ -10,4 +10,77 @@ compatibility.  We also mention cases where backwards compatibility was
...
+Please check [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) notice for details.
+
+### Feature Gaps between Telemetry V2 and Mixer Telemetry
+
+* Out of mesh telemetry is not supported. Some telemetry is missing if source or the destination of traffic is not sidecar injected.
+* Egress gateway telemetry is [not supported](https://github.com/istio/istio/issues/19385).
+* TCP telemetry is only supported with `mtls`.
+* Black Hole telemetry for TCP and HTTP protocols is not supported.
+* Histogram buckets are [significantly different](https://github.com/istio/istio/issues/20483) than Mixer Telemetry and cannot be changed.
+
```
Comment (myidpt) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion

## Istio workload key and certificate provisioning

* We have stabilized the SDS certificate and key provisioning flow. Now the Istio workloads are using SDS to provision certificates. The secret volume mount approach is deprecated.
* Please note when mutual TLS is enabled, Prometheus deployment needs to be manually modified to monitor the workloads. The details are described in this [issue](https://github.com/istio/istio/issues/21843). This is not required in 1.5.1.

```

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 22]
```diff
@@ -10,4 +10,82 @@ compatibility.  We also mention cases where backwards compatibility was
...
+Note: despite the naming, in Istio 1.5 when `controlPlaneSecurityEnabled` is set to `false`, communication between the control plane will be secure by default.
+
+## Multicluster setup
+
+{{< warning >}}
+We recommend that you **do not upgrade** to Istio 1.5.0 if you are using a multicluster setup.
+
+Istio 1.5.0 multicluster setup has several known issues ([27102](https://github.com/istio/istio/issues/21702), [21676](https://github.com/istio/istio/issues/21676)) that make it unusable in both shared control plane and replicated control plane deployments. These issues will be resolved in Istio 1.5.1.
+{{< /warning >}}
```
Comment (linsun) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
## Helm upgrade
If you used `helm upgrade` to update your cluster to newer Istio versions, we recommend you to switch to use [`istioctl upgrade`](docs/setup/upgrade/istioctl-upgrade/) or follow the [helm template](/docs/setup/upgrade/cni-helm-upgrade/) steps. 
```

**Comment by linsun** [comment]
nice job @howardjohn - thanks for putting this together

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 23]
```diff
@@ -10,4 +10,86 @@ compatibility.  We also mention cases where backwards compatibility was
...
+
+## Helm upgrade
+
+If you used `helm upgrade` to update your cluster to newer Istio versions, we recommend you to switch to use [`istioctl upgrade`](docs/setup/upgrade/istioctl-upgrade/) or follow the [helm template](/docs/setup/upgrade/cni-helm-upgrade/) steps.
```
Comment (fpesce) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
If you used `helm upgrade` to update your cluster to newer Istio versions, we recommend you to switch to use [`istioctl upgrade`](/docs/setup/upgrade/istioctl-upgrade/) or follow the [helm template](/docs/setup/upgrade/cni-helm-upgrade/) steps.
```
